src/app/containers/Ad/Amp/index.jsx

```diff
@@ -106,18 +100,15 @@ const AmpAd = ({ slotType }) => {
         {AMP_ACCESS_JS}
         {AMP_ACCESS_FETCH(service)}
       </Helmet>
-      <AdSection
-        aria-label={ariaLabel}
-        role="region"
-        data-e2e="advertisement"
-        aria-hidden="true"
-      >
-        <AdContainer slotType={slotType}>
-          <StyledWrapper>
-            <DisplayWrapper
-              amp-access="toggles.ads.enabled"
-              amp-access-hide="true"
-            >
+      <DisplayWrapper amp-access="toggles.ads.enabled" amp-access-hide="true">
```

Comment:
It's been a while... Ok, so here we're saying that we never render the placeholder on the server, and when we get whether or not ads are enabled back from the toggles API, we open up the space? Does this increase CLS? Does that matter on AMP?

Reply:
Is there an alternative? We had an issue before where the placeholder was showing even when we determined we were in the UK?

Reply:
This is probably more of a problem now actually. That was determined to be ok for world service, but I agree, that's pants for public service.

Reply:
@chris-hinds had a theory earlier on that when google index the page the placeholder's presence may be cached because it will have a cache sharded by geolocation. Will have to see if that happens in live and maybe come back to this?

Reply:
> @chris-hinds had a theory earlier on that when google index the page the placeholder's presence may be cached because it will have a cache sharded by geolocation. Will have to see if that happens in live and maybe come back to this?

yeah this is just a theory though :/